sentry-config.cmake.in

```diff
@@ -1,34 +1,27 @@
 @PACKAGE_INIT@
-include(CMakeFindDependencyMacro)
 
 set(SENTRY_BACKEND @SENTRY_BACKEND@)
 set(SENTRY_TRANSPORT @SENTRY_TRANSPORT@)
 set(SENTRY_BUILD_SHARED_LIBS @SENTRY_BUILD_SHARED_LIBS@)
 set(SENTRY_LINK_PTHREAD @SENTRY_LINK_PTHREAD@)
```

Comment:
Note that in the following lines, there is now a mix of variables exported explicitly in this way, and variables expanded directly via `"@VAR@"`.